bertinNDAYISHIMIYE/capstoneProject-MyBrand
# ft-enquiries-test

###What does this PR do?
>add enquiry test logic

###Description of the task to be completed!
>Test the enquiry endpoints

###How should this be tested manually?
>Clone this repo with: https://github.com/bertinNDAYISHIMIYE/capstoneProject-MyBrand
>Checkout to ft-updateBlog branch
>Make sure node.js is installed
>Run command npm init for npm initialisation

###Any background context you want to provide?

N/P
###Relavant trello card
https://trello.com/c/1s2ELBfS
